notifications/http.go

```diff
@@ -20,19 +20,23 @@ type httpSink struct {
 	client    *http.Client
 	listeners []httpStatusListener
 
+	ignoreActions []string
+
 	// TODO(stevvooe): Allow one to configure the media type accepted by this
 	// sink and choose the serialization based on that.
 }
 
 // newHTTPSink returns an unreliable, single-flight http sink. Wrap in other
 // sinks for increased reliability.
-func newHTTPSink(u string, timeout time.Duration, headers http.Header, transport *http.Transport, listeners ...httpStatusListener) *httpSink {
+func newHTTPSink(u string, timeout time.Duration, headers http.Header, transport *http.Transport, iActions []string, listeners ...httpStatusListener) *httpSink {
```

Comment:
Sinks are compositional and should be focused on a single goal. Rather than adding an argument here, fix `newIgnoredMediaTypesSink` to handle this.

Reply:
Thanks，I will update the commit and submit it later. O(∩_∩)O